bin/fuel-core/src/cli/run.rs

```diff
@@ -714,6 +714,8 @@ impl Command {
                 max_tx_update_subscriptions: tx_number_active_subscriptions,
                 subscription_ttl,
                 status_cache_ttl: status_cache_ttl.into(),
+                // TODO: Where can I get this value?
+                protocol_public_key: Default::default(),
```

Comment:
You can get it from the chain config like we do during verification of block's fields

Reply:
<img width="766" alt="image" src="https://github.com/user-attachments/assets/5c9b81f8-4703-4b33-b036-e9b8c57afedc" />